src/plugins/tonegen/tone.c

```diff
@@ -141,39 +146,60 @@ struct tone *tone_create(struct stream *stream,
 
 void tone_destroy(struct tone *tone, bool kill_chain)
 {
+    if( !tone )
```

Comment:
nit: different formatting here, and some later as well

Reply:
fixed, hopefully all of them